source/codegen/metadata/nifake/functions.py

```diff
@@ -101,10 +101,10 @@
         ],
         'returns': 'ViStatus'
     },
-    'BoolArrayOutputFunction': {
+    'BoolArrayInputFunction': {
```

Comment:
[Note] This diff is hard to read, but I just moved the functions around to have them sorted. The best way to see this is looking at the updates in nifake.proto to see the new ordering of service methods.

I also removed the `'codegen_method': 'no'` methods as they weren't being used in generation and I am removing them in the hapigen output. I figured if we want to add them back we'll add them through the hapigen metadata / plugin. I don't think having any of them in grpc-device metadata is useful. It also will help aligning the functions.py metadata diffs when we update ni-fake metadata from hapigen output.